### ISSUE TITLE
Pending on my vivawallet account! 

### ISSUE THREAD

**Issue by Barnaby-Clifton** [VivaPayments/API#1321]
Why is my website currently on 'pending'? 

greendragonstores.co.uk

**Comment by matthewrgourd** [comment]
Hi @Barnaby-Clifton this is usually because some additional information is needed before we can activate a new payment source. Have you received any emails from our account team?

**Comment by Barnaby-Clifton** [comment]
No, it's all validated and my api is set up properly. I am FLUMMOXED as to why this is happening.

**Comment by matthewrgourd** [comment]
Ok, I will check your account then. Please send me your Merchant ID but via email (for security): mgourd@vivawallet.com. Reference #1321 so I can map to this issue.

**Comment by Barnaby-Clifton** [comment]
39245731 is my wallet ID.

I hope that's sufficient

**Comment by matthewrgourd** [comment]
I checked your account just now and apparently we sent you an email on 2 Dec requesting further info. This is required before we can make the payment source active. Did you receive the message?

**Comment by Barnaby-Clifton** [comment]
What info? I have sent you everything that I needed to. It was about the
verification documentation.

**Comment by matthewrgourd** [comment]
Hi @Barnaby-Clifton on 2 Dec we sent you three points that needed addressing before the payment source could be enabled. Didn't you receive this? Item 1 related to terms, 3 contact details, and 3 HTTPS status on your website.

**Comment by Barnaby-Clifton** [comment]
No, I can't find it. I thought that I gave you everything though.

**Comment by matthewrgourd** [comment]
Hi @Barnaby-Clifton , ok, I've asked our customer service department to send it through again. Apologies for any inconvenience caused.

**Comment by Barnaby-Clifton** [comment]
Uhhh, I think I set it all up before and it was running but sure.
I will take it up with them because I can't have these issues all the time
